workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,200 @@
+
+# Create New Widget
+
+Creating a new widget is useful for custom reporting and exploration. To create a new widget, navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
+
+![](https://lh7-us.googleusercontent.com/Qn2pnY24-2CuG7moVbGCi4UqO0aoIMdqNjsYaSipLHo3gfUOhD_z0vchbVyMNrN3zb6gPZeCAkkoJWmndzgqtcZVVFz9jzJYD_gkhP3qWNytupLefOGdMGLhd5Z9SjTvxqXbMmM1x6XYF0QzXK9LNn8)
+
+![](https://lh7-us.googleusercontent.com/Oe2a3IYXnQ4JTYXRbzMe9c7RZSHXFySFLJGFZCxWhJIFIvlNM2hOmGgQdCapl2h8Hk5-FhEWRwALQxZ1lJ2BGJtnvQnXA_r0TCqX5o3OeXwJnAqQrR1wkm77-uZLmL5V7QTiGpPr4Uhc6SRvjaO3Yjg)
+
+### Chart Types
+
+Choose from one of the following types of charts for your widget:
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+| __Chart Type__     | __Use Case__ |
+| :-------------     | :------------|
+| __Timeseries__     | Describes trends that are collected and ordered with respect to time  |
+| __Bar__            | Helpful for visually comparing differences categorical data and groups |
+| __List__           | Ideal for comparing numerical data associated with numerical data |
+| __Query Value__    | Helpful when you need to always know the status/value of a single metric|
+| __Pie Chart__      | Organizes and shows data as a percentage of a whole |
+| __Scatter Plot__   | Ideal for comparing the relationship/correlation between two metrics |
+| __Tree Plot__      | Displays hierarchal relationship of your data as a function of size and color |
+| __Top List Chart__ | Displays highest-ranking elements or categories based on predefined metric criteria for your most important metrics |
+| __Heatmap Chart__  | Helps you identify the range of values that your data resides with respect to time |
+| __Hexagon Chart__  | Ideal for identifying which host and containers are running in your environment |
+
+### Create Your Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within your query.
+
+For certain aggregation types, Average, Sum, Min and Max, there is a concept of an __Aggregate By__. To understand how __Group By__ and __Aggregate By__ work together, think of your queries like 2-dimensional arrays, where __Aggregate By__ is a function that performs a transformation on each subarray and __Group By__ performs an aggregate function on the aggregate outputs across all arrays. 
+
+![](https://lh7-us.googleusercontent.com/8sltD2VI-oNKeR0PPF2cp1rx5ktjK-KfXnF-vOUzvfXbcDElFeTKEXmsx6wm3PDN95xl1IjbcOwyX9Oq5CXoKVcjroG1V0eq3JeaL0XP8cvUU4yqIKKyFQa2t7dDrpSJJKaM6emthaYHDvGNSr3ztOs)
+
+
+#### Group By
```

Comment:
Idea: we can rename this to "X By" or "Function By", because it's not always "Group", technically, it can also be average/min/max/etc. That will also help with some of the wording in this doc. WDYT?